src/components/ui/icons/PoweredByMorpho.tsx

```diff
@@ -1,19 +1,52 @@
 "use client";
```

Comment:
See comment on the deleted images, Morpho asked us to use their actual brand assets here (sry, designs don't have).